Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "vector"
-version = "0.21.1"
+version = "0.21.2"
```

Comment:
Premature?

Reply:
Ah, you're right, I should bump this after tagging.